### PR TITLE
Feature/UI label atelier

### PR DESCRIPTION
**Pull request by Deepesh-emproto**

### CODE REVIEW

File: src/backend/HomeAtelier.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import Demo from '../Assets/jayporeImage.jpg';
+
+export type atelierDatatype ={key:number;image:string}[]
```
Comment (prateekoli9) [inline src/backend/HomeAtelier.tsx:3]: Every component will not have a backend file. One backend file will be present for a feature. For all homepage related call, use home.ts backend(Already in development branch)

File: src/components/Label-Atelier/label-atelier.module.scss [hunk 2]
```diff
@@ -0,0 +1,124 @@
+@use '../../styles/config.scss' as *;
+
+
+.Label_Atelier_Parent{
```
Comment (prateekoli9) [inline src/components/Label-Atelier/label-atelier.module.scss:4]: use camelCase (labelAtelier)

File: src/components/Label-Atelier/label-atelier.module.scss [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    flex-direction: column;
+
+    &_Text{
+       margin-bottom: 10px;
+       justify-content: center;
+       display: flex;
+    }
+    
+}
+.Label_Atelier_Parent_Slider{
```
Comment (prateekoli9) [inline src/components/Label-Atelier/label-atelier.module.scss:19]: use scss syntax, put this inside the first parent class using &_slider syntax

File: src/components/Label-Atelier/label-atelier.module.scss [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        .slick-list{
+            width: 100%;
+        }
+       
+    }
+}
+.slick-prev{
+    background-color: black !important;
+}
+.Label_Atilier_Parent_Text_Line{
```
Comment (prateekoli9) [inline src/components/Label-Atelier/label-atelier.module.scss:63]: same comment as above for these classes

File: src/backend/HomeAtelier.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+export const AtelierArray:atelierDatatype =[
```
Comment (prateekoli9) [inline src/backend/HomeAtelier.tsx:5]: Don't export  this directly, export a function that returns this array

File: src/components/Label-Atelier/label-atelier.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+              <div className={styles.Label_Atelier_Parent_Text_Color}><b>JAYPORE OWN LABEL ATELIER</b></div>
+              <div className={styles.Label_Atilier_Parent_Text_Line}></div>
+          </div>
+          <Slider className={styles.Label_Atelier_Parent_Slider} {...settings}>
+             {
+                 AtelierArray.map((val):JSX.Element=>{
+                  return (
+                     <>
+                      <CardComponent images={val.image} height={'100%'} width={'95%'} />
+                      {console.log(val.image)}
```
Comment (prateekoli9) [inline src/components/Label-Atelier/label-atelier.tsx:62]: Remove console.log

File: src/components/Label-Atelier/label-atelier.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+};
+
+const PrevArrow: FC<ArrowProps> = ({ onClick }): JSX.Element => {
+    return (
+        <div className={styles.prevArrow} onClick={onClick}>
+            <MdNavigateBefore className={styles.NextArrowButton}/>
+        </div>
+    );
+};
+    const settings:setObject = {
```
Comment (prateekoli9) [inline src/components/Label-Atelier/label-atelier.tsx:38]: fix indentation

File: src/pages/home/home.component.tsx [hunk 8]
```diff
@@ -16,17 +18,20 @@ function HomeComponent(): JSX.Element {
...
+  return (
         <div className={styles.home}>
             <div className={styles.home__header}>
+                <div className={styles.OfferBanner_position}>
+                  <OfferBanner/>
+                </div>
                 <div className={styles.home__header__links}>
-                    <OfferBanner />
-                    <Header />
-                    <MainSwiper />
```
Comment (prateekoli9) [inline src/pages/home/home.component.tsx:25]: Why others' components removed?

File: src/pages/home/home.module.scss [hunk 9]
```diff
@@ -45,3 +45,16 @@
         }
     }
 }
+.Homepage_Component_Coursel{
```
Comment (prateekoli9) [inline src/pages/home/home.module.scss:48]: use camelCase

**Comment by Deepesh-emproto** [comment]
Screenshot from 2022-09-12 12-53-59

File: src/backend/home.ts [hunk 10]
```diff
@@ -167,6 +168,39 @@ export function getCarouselImages(): object[] {
     return imageCollection;
 }
 
+
+export type atelierDatatype ={key:number;image:string}[]
+
+export const homePageAtelierPart=():atelierDatatype=>{
```
Comment (prateekoli9) [inline src/backend/home.ts:174]: rename to getAtelierData

File: src/backend/home.ts [hunk 11]
```diff
@@ -167,6 +168,39 @@ export function getCarouselImages(): object[] {
     return imageCollection;
 }
 
+
+export type atelierDatatype ={key:number;image:string}[]
```
Comment (prateekoli9) [inline src/backend/home.ts:172]: rename to atelierDataType (camel case)

**Pull request merged**